app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less

```diff
@@ -90,3 +90,19 @@
         }
     }
 }
+
+//
+//  Tablet
+//  _____________________________________________
+@media only screen and (max-width: @screen__m) {
+  .footer .block.newsletter {
+    width: 42%;
+  }
+}
+
+
```

Comment:
Please, remove the extra empty lines. It's enough to have just one empty line at the end. Thank you

Reply:
Hi @rogyar ,
I have remove extra line from app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less
Thanks,
Dipti